vmss-prototype/vmss-prototype

```diff
@@ -729,26 +730,29 @@ def vmss_prototype_update(sub_args):
 
                 # Delete the instance to prevent idns side-effects from future VMs coming online based on its prototype
                 # See https://github.com/jackfrancis/kamino/issues/26
-                run(az(['vmss', 'delete-instances'], subscription) + [
+                ret = run(az(['vmss', 'delete-instances'], subscription) + [
```

Comment:
Is this the correct way to capture this? I'm afraid we're introducing more blocking serialization than we want